styles.css

```diff
@@ -87,3 +88,249 @@ body {
   padding-left: 10px;
   padding-right: 10px;
 }
+
+.Middle-page {
+  background-color: #e5e5e5;
+  height: 100%;
+  margin-top: -35px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.Middle-page > h1 {
+  text-align: center;
+  padding-top: 100px;
+  padding-bottom: -10px;
+  font-size: 40px;
+  font-family: 'Crete Round', serif;
+}
+
+.stand-alone {
+  padding-top: 50px;
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  color: #091e42;
+  margin-left: -5px;
+  margin-right: -5px;
+}
+
+.stand-alone-head {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  color: #091e42;
+  padding-top: -10px;
+  margin-bottom: -10px;
+}
+
+.stand-alone-p {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  color: #091e42;
+  padding-top: -10px;
+}
+
+.buttons {
+  display: flex;
+  margin-left: -14%;
+}
+
+.button-text {
+  padding: 10px 15px;
+  list-style: none;
+  border: 1px solid;
+  font-size: 15px;
+  font-family: 'Inter', sans-serif;
+  line-height: 24px;
+  margin: 5px;
+}
+
+.button-text > a {
+  text-decoration: none;
+}
+
+.click {
+  background-color: #ff6b00;
+  color: #fff;
+  height: 40px;
+  width: auto;
+  padding: -10px;
+  margin-top: -10px;
+  border: none;
+}
+
+.click:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  box-sizing: border-box;
+}
+
+.grid {
```

Comment:
- [ ] It would be better if the content of each of the projects is aligned/positioned with `flexbox`